### ISSUE TITLE
Figure out NSI data to fit with INCORE building inventory

### ISSUE THREAD

**Issue by ywkim312** [IN-CORE/pyincore-data#53]
The NSI data should be adjusted to fit into INCORE building inventory schema. 
There are several fields are missing in NSI dataset and some fields have different field name from incore building inventory schema. 
Analysis the NSI attribute table and find out how to modify the attribute table.
1. Find out the fields that exist in NSI but have different name from incore building inventory
2. Find out missing fields in NSI that building inventory needs to have
3. Find out if the actual field format is correct (e.g. string, decimal digit, and so on)
4. Find out the way of converting the data value if the field exist with the values with different expression or different unit.

Please use the attached shapefile to check the NSI attribute. If there is any missing 

[joplin.zip](https://github.com/IN-CORE/pyincore-data/files/9803989/joplin.zip)

The metadata for the fields are in here 
https://www.hec.usace.army.mil/confluence/nsi/technicalreferences/latest/technical-documentation

**Comment by longshuicy** [comment]
Same is it related to #96?

**Comment by ywkim312** [comment]
> Same is it related to [#96](https://github.com/IN-CORE/pyincore-data/issues/96)?

Yes that is very old one for figuring out and this one got cancelled. Let me close it
